unattended_scripts/install_functions/opendistro/elasticsearch.sh

```diff
@@ -79,6 +79,7 @@ configureElasticsearchAIO() {
 
     eval "cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ ${debug}"
     eval "./securityadmin.sh -cd ../securityconfig/ -icl -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin-key.pem ${debug}"
+    eval "cd - ${debug}"
```

Comment:
Remove this line

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/1046/commits/cb80bbeb99ab7a9aee91d27be8dd4004610c0b10